spinnaker_components/ensemble/ensemble.h

```diff
@@ -65,10 +58,11 @@ typedef struct ensemble_parameters {
   uint machine_timestep;   //!< Machine time step  / useconds
 
   uint t_ref;              //!< Refractory period \f$\tau_{ref} - 1\f$ / steps
-  value_t dt_over_t_rc;    //!< \f$\frac{dt}{\tau_{rc}}\$
+  value_t exp_dt_over_t_rc;    //!< \f$-\exp{\frac{dt}{\tau_{rc}}}\$
 
   current_t *i_bias;        //!< Population biases \f$1 \times N\f$
-  neuron_status_t *status;  //!< Neuron status
+  value_t *neuron_voltage;  //!< Neuron voltages
+  uint8_t *neuron_refractory;  //!< Refractory states
```

Comment:
The bit-field that was there before with the 28-bits and the 4-bits - My
comment was more a late-to-the-party affirmation of your change than
anything else!

On 19 June 2015 at 13:18, Andrew Mundy <notifications@github.com> wrote:

> In spinnaker_components/ensemble/ensemble.h
> <https://github.com/project-rig/nengo_spinnaker/pull/35#discussion_r32823486>
> :
>
> >
> >    current_t *i_bias;        //!< Population biases \f$1 \times N\f$
> > -  neuron_status_t *status;  //!< Neuron status
> > +  value_t *neuron_voltage;  //!< Neuron voltages
> > +  uint8_t *neuron_refractory;  //!< Refractory states
>
> Sorry, @neworderofjamie <https://github.com/neworderofjamie>, which
> structure? I'm assuming that the two arrays is fine?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/project-rig/nengo_spinnaker/pull/35/files#r32823486>.
>